Content.Shared/Roles/DepartmentPrototype.cs

```diff
@@ -47,6 +48,18 @@ public sealed partial class DepartmentPrototype : IPrototype
     /// </summary>
     [DataField]
     public bool EditorHidden;
+
+    /// <summary>
+    /// RMC14 for logical communication of the department and its radio channel.
+    /// </summary>
+    [DataField]
+    public ProtoId<RadioChannelPrototype>? DepartmentRadio { get; private set; }
```

Comment:
This also needs to be moved to our own partial of DepartmentPrototype, its in Content.Shared/_RMC14/Marines/Roles